### PR TITLE
TM-1263 get the obc ids for the dynamic apis

### PR DESCRIPTION
**Pull request by BaldwinJordan**
Nothing has been done yet to fix the duplicate/missing values, but they dynamic api should work

### CODE REVIEW

**Comment by mjoyce91** [comment]
@BaldwinJordan would we be able to add these to the projected vacancy endpoint as well?

File: talentmap_api/fsbid/services/available_positions.py [hunk 1]
```diff
@@ -208,8 +208,8 @@ def fsbid_ap_to_talentmap_ap(ap):
                 "id": None,
                 "code": ap.get("pos_location_code", None),
                 "tour_of_duty": ap.get("tod", None),
-                "post_overview_url": None,
-                "post_bidding_considerations_url": None,
+                "post_overview_url": services.get_post_overview_url(ap.get("pos_location_code", None)),
+                "post_bidding_considerations_url": services.get_post_bidding_considerations_url(ap.get("pos_location_code", None)),
```
Comment (mjoyce91) [inline talentmap_api/fsbid/services/available_positions.py]: :+1:

Comment (BaldwinJordan) [reply talentmap_api/fsbid/services/available_positions.py]: Jeesh so demanding :)

**Comment by BaldwinJordan** [comment]
Did I finally get it all?

**Pull request merged**